libpod/runtime.go

```diff
@@ -739,11 +742,42 @@ func getLockManager(runtime *Runtime) (lock.Manager, error) {
 	return manager, nil
 }
 
+// probeConmon calls conmon --version and verifies it is a new enough version for
+// the runtime expectations podman currently has
+func probeConmon(conmonBinary string) error {
+	cmd := exec.Command(conmonBinary, "--version")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
+	r, err := regexp.Compile(`^conmon version (?P<Major>\d+).(?P<Minor>\d+).(?P<Patch>\d+)`)
```

Comment:
Should this be a `MustCompile` so we compile error instead of runtime error when the regex fails?

Reply:
https://golang.org/pkg/regexp/#MustCompile panics if it doesn't match in runtime. Seems a lot more unpleasant than an error message

Reply:
nevermind, you are right